src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-types/lancie-admin-ticket-options.html

```diff
@@ -0,0 +1,164 @@
+        </lancie-form>
+      </div>
+      <div class="dialog-actions">
+        <paper-button on-tap="tryAddOption">Add</paper-button>
+      </div>
+    </lancie-dialog>
+
+    <lancie-dialog id="deleteTicketOptionDialog">
+      <h2>Confirm deletion</h2>
+      <div class="dialog-content">
+        <lancie-error id="dialogError"></lancie-error>
+        <h4>Are you certain you want to delete this ticket option?</h4>
+        <p>[[deleteOption.name]]</p>
+      </div>
+      <div class="dialog-actions">
+        <paper-button dialog-dismiss>Cancel</paper-button>
+        <paper-button on-tap="tryOptionDelete" class="confirm-delete-button">Delete</paper-button>
+      </div>
+    </lancie-dialog>
+  </template>
+  <script>
+    (function () {
+      'use strict';
+
+      Polymer({
+        is: 'lancie-admin-ticket-options',
+        properties: {
+          data: Object,
+          mappings: {
+            type: Object,
+            value: function() {
+              return {
+              };
+            }
+          },
+          renderers: {
+            type: Object,
+            value: function() {
+              return {
+                possibleOptions: function(data) {
+                  var text = data.reduce(function(text, t){
+                    return text + t.name + ' ' + t.price +', ';
+                  }, '');
+                  return text.substr(0, text.length - 2);
+                },
+              };
+            }
+          }
+        },
+
+        tryAddOption: function(e, request) {
+          this.$.addTicketOptionForm.validateAndSubmit();
+          this.$.addTicketOptionForm.reset();
+        },
+
+        openAddOptionDialog: function() {
+          this.$.addTicketOptionForm.reset();
+          this.$.addTicketOptionDialog.open();
+        },
+
+        onAddOptionResponse: function(e, request) {
+          if (request.succeeded) {
+            this.$.addTicketOptionDialog.close();
+            this.$.getOptionAjax.generateRequest();
+          } else {
+            this.$.addDialogError.setError("Could not add option. Please refresh and try again.")
+          }
+        },
+
+        deleteTicketOptionDialog: function(event) {
+          this.deleteOption = this.options.filter(option => {
+            return option.id === event.detail.id;
+          })[0];
+          this.$.deleteTicketOptionDialog.open();
+        },
+
+        tryOptionDelete: function() {
+          this.$.ajaxDeleteTicketOption.generateRequest();
+        },
+
+        onOptionDeleteResponse: function(e, request) {
+          if (request.succeeded) {
+            this.$.deleteCommitteeMemberDialog.close();
+            this.$.getOptionAjax.generateRequest(); // Ensures we have a mirror of the latest server state
+          } else {
+            this.$.dialogError.setError('Something went wrong! Please refresh and try again.')
+          }
+        },
+
+        onOptionResponse: function(e, request) {
```

Comment:
We can change to an early return here, too.

Don't forget your `;`'s

As a sidenote, we shouldn't need the `on-response` event, this should be the `on-lancie-form` event. If you want to make a small change which makes the code a bit more readable, look at this issue: https://github.com/AreaFiftyLAN/lancie-form/issues/6

Reply:
I did not edit `lancie-form` to change the event name

Reply:
I know you didn't edit it, I was just making a remark on how the lancie-form is now different from lancie-ajax. ;)

Reply:
But right now for `lancie-form` the event is still called `on-response`, so not something I can change right now